evalml/tests/preprocessing_tests/test_balanced_classification_sampler.py

```diff
@@ -167,13 +170,13 @@ def test_classification_imbalanced_normal_imbalance_binary(min_samples, balanced
     indices = bcs.fit_resample(X, y)
     X2 = X.loc[indices]
     y2 = y.loc[indices]
-    if balanced_ratio >= 6:
+    if balanced_ratio < .2:
```

Comment:
Same as before, please change to 0.25 for clarity, and I see one more similar usage below.

Reply:
Similar reasoning as before!